superset/charts/commands/importers/v1/__init__.py

```diff
@@ -89,9 +89,7 @@ def _import(
                 config.update(
                     {
                         "datasource_id": dataset.id,
-                        "datasource_type": "view"
-                        if dataset.is_sqllab_view
-                        else "table",
+                        "datasource_type": "table",
```

Comment:
I am not familiar with this part, but the `datasource_type = view` seems the correct value here. Here is the schema [defination](https://github.com/apache/superset/blob/22a92ed72278d45049cc33eba5219d215625e4d9/superset/charts/schemas.py#L199-L203) for updating chart.

@betodealmeida could you take a look?

Reply:
but virtual dataset should have `view` as a datasource_type?

Reply:
I'm also not familiar with this part, so I created a chart from a virtual dataset and it has `datasource_type` set to `table`. Looking at the source code it seems that "view" here means a database `VIEW`, not a virtual dataset, and is only used by the DB engine spec to retrieve the list of views from a given database:

https://github.com/apache/superset/blob/4a835a4299bbe90def232e376f919bc494b2d0a1/superset/db_engine_specs/base.py#L873-L879

So I think this should be `table`, and we also need to fix this line:

https://github.com/apache/superset/blob/4a835a4299bbe90def232e376f919bc494b2d0a1/superset/commands/importers/v1/examples.py#L141